### PR TITLE
More rephrasing, suggestions (part 2)

### PR DESCRIPTION
**Pull request by zmousm**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -671,11 +671,12 @@
   - **`git tag`**
 - Checkout to a specific tag
   - **`git checkout <tag_name>`**
+  - this puts you in a detached HEAD state
 - Pushing tags to the remote
   - **`git push <push_options_and_branches> --tags`**
 - Delete tag
   - **`git tag -d <tag_name>`**
-- Change where a tag points
+- Change the commit pointed at by a tag
```
Comment (ioparaskev) [inline index.html]: maybe "Change the commit a tag points to" is better?

File: index.html [hunk 2]
```diff
@@ -817,49 +821,50 @@
...
-  - Loses the context provided by a merge commit—you can’t see when upstream changes were incorporated into the feature
-  - Can be catastrophic if done on public branches & history
+  - Rebasing loses the context provided by a merge commit—you won’t see how (when) distinct sets of changes were brought together
+  - Rebasing can be catastrophic if done on public branches & history
   - **`git pull --rebase`** to *unlock* this feat or **`git config --global branch.autosetuprebase always`**
 
 ---
 class: middle, center
-# destructive rebase illustrated
+# destructive rebase illustrated<br>.center.remark-code[master branch rebased on feature]
```
Comment (ioparaskev) [inline index.html]: I made a change to the svg so I think this is not needed anymore

**Review by ioparaskev — CHANGES_REQUESTED** [review]
thanks again :]

File: index.html (hunk 1 above)
Comment (zmousm) [reply index.html]: OK

File: index.html (hunk 2 above)
Comment (zmousm) [reply index.html]: OK

**Pull request merged**